pkg/provider/resource_pgd.go

```diff
@@ -331,6 +331,11 @@ func PgdSchema(ctx context.Context) schema.Schema {
 								},
 							},
 						},
+						"ro_connection_uri": schema.StringAttribute{
+							MarkdownDescription: "Cluster read-only connection URI. Only available for high availability clusters.",
```

Comment:
remove `Only available for high availability clusters`

Reply:
Fixed in the latest commit.